src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -1063,6 +1066,16 @@ public ProcessExecutionResults RunApkInstrumentation(string apkName, string? ins
         return result;
     }
 
+    private void DisableChatty()
+    {
+        var result = RunAdbCommand(new[] { "shell", "logcat", "-P", "'\"\"'" }, TimeSpan.FromMinutes(1));

```

Comment:
is the double-quotes-in-single-quotes really necessary?
you also don't need `shell` since `logcat` is (also) a comment on `adb`.

Reply:
Yeah, it fails otherwise

Reply:
did you try without the `shell`? I think it might be because of that

Reply:
Yeah, it behaves the same way.